LimeBean/BeanApi.cs

```diff
@@ -573,6 +573,10 @@ public partial class BeanApi : IDisposable, IBeanCrud, IBeanFinder, IDatabaseAcc
             set { Storage.ConvertEmptyStringToNull = value; }
         }
 
+        /// <summary>
+        /// Gets or Sets whether integers are detected and extracted if in string type variables, when storing to the database.
+        /// Query paramaterisation and the database still handle storage types
+        /// </summary>
         public bool RecognizeIntegers {
```

Comment:
@AlekseyMartynov do you recall the purpose of this flag in `DatabaseStorage.ConvertValue(...)`? 
It seems to me that converting an integer stored in a string variable to an integer before parametising, wouldn't have any positive or negative effect on the DbCommand, since the database would recognise the destination field and attempt conversion? Or would conversion fail from string->int but not int->string?

Anyway, just a point of interest so I get documentation right, and understand for future reference.